morf-testsupport/src/main/java/org/alfasoftware/morf/xml/DummyXmlOutputStreamProvider.java

```diff
@@ -77,7 +77,7 @@ public void clearDestination() {
    * @return Convert the output to a String
    */
   public String getXmlString() {
-    return new String(testOutputStream.toByteArray());
+    return testOutputStream.toString();

```

Comment:
Are we sure about this?
To bytes and back was potentially doing some character encoding conversions.
Perhaps this is better? Perhaps this is a functional change?

Reply:
Reverted